### PR TITLE
Soft failure 1195046

### PR DESCRIPTION
**Pull request by skliu**
Quick fix for https://openqa.suse.de/tests/10657979

**Verification run:**

- https://openqa.suse.de/tests/10658934

### CODE REVIEW

File: lib/migration.pm [hunk 1]
```diff
@@ -35,6 +35,11 @@ our @EXPORT = qw(
 sub setup_sle {
     select_console 'root-console';
 
+    if (is_ppc64le && is_sle('<=12-sp5')) {
+        record_soft_failure('bsc#1195046 - ncurses display a wrong checker board character');
```
Comment (lemon-suse) [inline lib/migration.pm:39]: For me, I think we can see this as a test requirement while not a workaround, so please remove the softfail.

File: schedule/yast/autoyast/autoyast_create_hdd_sdk_gnome.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ schedule:
   - '{{handle_reboot}}'
   - installation/first_boot
   - console/system_prepare
+  - '{{disable_screensaver}}'
```
Comment (lemon-suse) [inline schedule/yast/autoyast/autoyast_create_hdd_sdk_gnome.yaml:23]: I think you make it mixed with another PR, please clean it.

File: lib/migration.pm (hunk 1 above)
Comment (skliu) [reply lib/migration.pm]: Done.

File: schedule/yast/autoyast/autoyast_create_hdd_sdk_gnome.yaml (hunk 2 above)
Comment (skliu) [reply schedule/yast/autoyast/autoyast_create_hdd_sdk_gnome.yaml]: Fixed, but I don't why there are two commits, I've rebased it.

**Review by lemon-suse — APPROVED** [review]

File: lib/migration.pm (hunk 1 above)
Comment (jknphy) [reply lib/migration.pm]: It was solved for sle 12 sp5, please keep the workaround for sle 12 sp4 only.
https://bugzilla.suse.com/show_bug.cgi?id=1205290#c3

Comment (skliu) [reply lib/migration.pm]: Done.
